DirectProgramming/C++/CompilerInfrastructure/OpenMP_Offload_Features/src/user_defined_mapper.cpp

```diff
@@ -0,0 +1,57 @@
+//==============================================================
+// Copyright © 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+// =============================================================
+
+extern "C" int printf(const char *, ...);
+struct C {

```

Comment:
Better to use a more descriptive name then "struct C".    This get's confused with the extern "C".